app/instances/spcoll/spcoll_config.rb

```diff
@@ -0,0 +1,9 @@
+# spcoll_config.rb
+
+# Application
+AppConfig[:enable_public] = false
+AppConfig[:enable_oai] = false
+AppConfig[:enable_docs] = false
+
+# Excel import setting
+AppConfig[:hide_do_load] = false
```

Comment:
I discussed with Monique the possibility of just nesting these under `if` clauses in the base config file and installing all the plugins in a base file. She also thinks the current approach is OK, but thought I would raise it in case you (Greg) felt strongly either way.